packages/meditrak-app-server/src/middleware/checkAppVersion.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2022 Beyond Essential Systems Pty Ltd
+ */
+
+import { NextFunction, Request, Response } from 'express';
+
+export const checkAppVersion = async (req: Request, res: Response, next: NextFunction) => {
+  try {
+    const { appVersion } = req.query;
+    if (!appVersion) {
+      throw new Error('appVersion unspecified, please upgrade your app');
```

Comment:
Nice! And this just logs the user out and shows them the error message on their login screen hey? Do you know if that is supported in the old versions that the users getting this will be using? 😅

Reply:
I'll test this before I merge! 👍